src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -122,11 +123,12 @@ def test_edit(self):
         assert unicode(addon.name) == data['name']
         assert addon.name.id == old_name.id
 
-        assert unicode(addon.slug) == data['slug']
         assert unicode(addon.summary) == data['summary']
 
         if self.listed:
             assert [unicode(t) for t in addon.tags.all()] == sorted(self.tags)
+            # We randomize slugs for unlisted add-ons
+            assert unicode(addon.slug) == data['slug']
```

Comment:
Not sure why this test changed to not check for slug with unlisted-only addons - we still show the addon slug for addons without listed versions, allow it to be edited, and it shoudn't change otherwise.

Reply:
Right, we do. I think I wanted to avoid a `assert len(data['slug']) == 20` for random slugs